db/user_variables.rb

```diff
@@ -17,40 +17,53 @@ class SeedUserVariables
     childcare_week_3: { value_type: :money, code: :CCWK4,  value: 1_00, description: 'Week 4 Cru Kids' },
-    childcare_week_4: { value_type: :money, code: :CCWKSC, value: 1_00, description: 'Staff Conference Cru Kids' },
+    childcare_week_4: { value_type: :money, code: :CCWKSC, value: 1_00, description: 'Kids Care Tuition Staff Conference' },
+
+    #Childcare costs by Type (NEW)
+    CARESC: { value_type: :money, code: :CARESC, value: 1_00, description: 'Kids Camp tuition during Staff Conference' },
+    CAMPSC: { value_type: :money, code: :CAMPSC, value: 1_00, description: 'Kids Care tuition during Staff Conference' },
 
     childcare_deposit: { value_type: :money, code: :CCNRF, value: 1_00,
       description: 'This is the non-refundable registration fee for any child' \
                    ' registered for either childcare or JrSr High programs.' },
 
-    # Jr/Sr Weekly Costs
+    # Jr/Sr Weekly Costs (OLD METHOD)
     junior_senior_week_0: { value_type: :money, code: :JRSRWK1,  value: 1_00, description: 'Week 1 Junior Senior' },
     junior_senior_week_1: { value_type: :money, code: :JRSRWK2,  value: 1_00, description: 'Week 2 Junior Senior' },
     junior_senior_week_2: { value_type: :money, code: :JRSRWK3,  value: 1_00, description: 'Week 3 Junior Senior' },
     junior_senior_week_3: { value_type: :money, code: :JRSRWK4,  value: 1_00, description: 'Week 4 Junior Senior' },
-    junior_senior_week_4: { value_type: :money, code: :JRSRWKSC, value: 1_00, description: 'Staff Conference Junior Senior' },
+    #junior_senior_week_4: { value_type: :money, code: :JRSRWKSC, value: 1_00, description: 'Staff Conference Junior Senior' },
+
+    # Jr/Sr Weekly Costs (CruStu) (NEW)
+    junior_senior_week_4: { value_type: :money, code: :JRSRWKSC, value: 1_00, description: 'CruStu tuition during Staff Conference' },
 
-    # Hot Lunches
+    # Hot Lunches (OLD METHOD)
     hot_lunch_week_0: { value_type: :money, code: :HL1,  value: 1_00, description: 'Cost of the Week 1 Hot Lunches' },
     hot_lunch_week_1: { value_type: :money, code: :HL2,  value: 2_00, description: 'Cost of the Week 2 Hot Lunches' },
     hot_lunch_week_2: { value_type: :money, code: :HL3,  value: 4_00, description: 'Cost of the Week 3 Hot Lunches' },
     hot_lunch_week_3: { value_type: :money, code: :HL4,  value: 8_00, description: 'Cost of the Week 4 Hot Lunches' },
     hot_lunch_week_4: { value_type: :money, code: :HLSC, value: 16_00, description: 'Cost of the Staff Conference Hot Lunches' },
 
-    # Hot Lunch Start Dates
+    #Hot Lunchs by type now (NEW)
+    HLCARE: { value_type: :money, code: :HLCARE, value: 16_00, description: 'Hot Lunch for Kids Care' },
+    HLCAMP: { value_type: :money, code: :HLCAMP, value: 16_00, description: 'Hot Lunch for Kids Camp' },
+    HLCRUSTU: { value_type: :money, code: :HLCRUSTU, value: 16_00, description: 'Hot Lunch for Crustu' },
```

Comment:
Match capitalization from line 43?
```suggestion
    HLCRUSTU: { value_type: :money, code: :HLCRUSTU, value: 16_00, description: 'Hot Lunch for CruStu' },
```